src/utils/puns.ts

```diff
@@ -56,9 +83,22 @@ const ERROR_PUNS = [
   'You are battering FSH',
```

Comment:
I've wanted to remove this one for awhile. Could we get rid of it?

Reply:
No objection here. Let me do one more push.